validphys2/examples/mc_gen_report.md

```diff
@@ -1,4 +1,7 @@
 %CHORUSNB 100 replicas
+Mean table
+----------
+{@art_data_mean_table@}
```

Comment:
Because there is a huge image below that was making the computer in which I was running crash. But once it gets to creating the img everything that I've modified has already been checked.